.github/workflows/sdl_ci.yml

```diff
@@ -3,15 +3,19 @@ name: sdl CI
 on:
   workflow_dispatch:
   push:
-    paths-ignore:
-      - "**.md"
-      - "**.yml"
-      - '!**/sdl_ci.yml'
+    paths:
+      - 'vlib/**'
+      - 'thirdparty/**'
+      - 'cmd/tools/builders/**.v'
+      - '**/sdl_ci.yml'
+      - '!**.md'
   pull_request:
-    paths-ignore:
-      - "**.md"
-      - "**.yml"
-      - '!**/sdl_ci.yml'
+    paths:
+      - 'vlib/**'
+      - 'thirdparty/**'
+      - 'cmd/tools/builders/**.v'
+      - '**/sdl_ci.yml'
+      - '!**.md'
```

Comment:
We use `v shader sdl/examples/sdl_opengl_and_sokol` so `cmd/tools/vshader.v` should be added too.

Reply:
Thanks!